Acdp 224 (#7)
Actindo-AG/Documentation: ActindoWorkFlow/UserInterface/05a_QueueTypes.md

[!!Workflows](ActindoWorkFlow)

# Overview
*Workflows > Queue Types > Tab OVERVIEW*

![Queue types](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/Overview.png "[Queue types]")

> 

Reviewer: Der Link scheint nicht auf der richtigen Stelle zu landen (Datei schon, aber Abschnitt nicht). Vielleicht bei dir prüfen?

**Queue Type List**

- [KILL WORKER]   

>   Click this button to stop all workers in all processes that are currently executing a job.   

Author: ```suggestion
  Click this button to stop all workers that are currently executing actions in all processes.   
```

  For detailed information, see [Kill all workers](/ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md#kill-all-workers).

> 

Reviewer: Der Link scheint nicht auf der richtigen Stelle zu landen (Datei schon, aber Abschnitt nicht). Vielleicht bei dir prüfen?

Reviewer: Update: passiert auch mit anderen Links. Ich weiß nicht warum.

Author: bei mir klappt alles...

- ![Refresh](/Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
  Click this button to update the list of queue types.

- ![Columns](/Assets/Icons/Columns.png "[Columns]") Columns (x)   
  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.

- [x]     
  Select the checkbox to display the editing toolbar.

- [DELETE]   
  Click this button to delete the selected queue type. The *QueueType has been deleted* pop-up window is displayed. The *Default* queue type cannot be deleted.  
  For detailed information, see [Delete a queue type](/ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md#delete-a-queue-type).

- [KILL WORKER OF THIS QUEUE TYPE]   

>   Click this button to stop all workers of the selected process that are currently executing a job.    

Reviewer: ```suggestion
  Click this button to stop all workers of the selected queue type that are currently executing a job.    
```

>   Click this button to stop all workers of the selected process that are currently executing a job.    

Author: ```suggestion
Click this button to stop all workers that are currently executing an action of the selected queue type.   
```

For detailed information, see [Edit a queue type](/ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md#edit-a-queue-type).

- *ID*   
  Queue type identification number. The ID number is automatically assigned by the system when the queue type is created.

- *Name*   
  Queue type name.  

- *Number of workers*   
  Number of the workers assigned to the queue type.

- *Sort mode of actions*   
  Execution priority assigned to the queue type. The following options can be displayed:   
  - **Oldest executable action**
  - **Executable actions of oldest process**

- ![Add](/Assets/Icons/Plus01.png "[Add]") (Add)   
  Click this button to create a queue type. The *Create Queue Type* view is displayed.   
  For detailed information, see [Create a queue type](/ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md#create-a-queue-type).

## Create Queue Type

*Workflows > Queue Types > Tab OVERVIEW > Button Add*

![Create queue type](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/CreateQueueType.png "[Create queue types]")

- ![Back](/Assets/Icons/Back02.png "[Back]") (Back)   

>   Click this button to close the *Create Queue Type* view and return to the queue type list. All changes are rejected.

Reviewer: ```suggestion
  Click this button to close the *Create Queue Type* view and return to the queue type list. All changes are discarded.
```

Reviewer: Ich meine, es (discard) irgendwo im System gelesen zu haben...

Author: Ich habe überall reject in der Doku stehen. (nicht nur workflows)..

- [SAVE]   
  Click this button to save the new queue type.

- *Name*   
  Enter a name for the queue type.  

- *Number of workers*   
  Enter the number of workers for the queue type.

>   > [Info] The number of workers you can assign to queue types depends on the number of booked vCores. -- Formel einfügen --

Reviewer 2: insert formula

- *Execution Prio*   
  Click the drop-down list and select the execution priority for the queue type. The following options are available:
  - **Oldest executable action**: The oldest executable actions within all existing processes is executed first by the workers.
  - **Executable actions of oldest process**: The executable actions within the oldest process are executed first by the workers.

> ## Edit Queue Type

Reviewer 2: Doesn't explain that all workers have to be stopped prior to editing the number of workers

Author: only description of UI, no explanation to relations
